dumper/dumper.c

```diff
@@ -31,9 +31,7 @@ FOO Recipients;
 extern FOO Signer[];
 
 #ifdef USE_CBOR_CONTEXT
-#define CBOR_CONTEXT_PARAM , NULL
-#else
-#define CBOR_CONTEXT_PARAM
+#define context NULL
```

Comment:
shouldn't use here CAPITALS? I foresee issues here :D

Reply:
No, I really need to just declare the global variable.